onnx/defs/data_type_utils.cc

```diff
@@ -96,15 +97,19 @@ DataType DataTypeUtils::ToType(const TypeProto& type_proto) {
   std::lock_guard<std::mutex> lock(GetTypeStrLock());
   if (GetTypeStrToProtoMap().find(typeStr) == GetTypeStrToProtoMap().end()) {
     TypeProto type;
-    FromString(typeStr, type);
+    Common::Status st = FromString(typeStr, type);
+    if (!st.IsOK())
+      return nullptr;
```

Comment:
Can we print something in this case? Maybe st.ErrorMessage(..) or something like that?

Reply:
No. Unless onnx has a Logger interface, otherwise it's better to not print anything in onnx code. Because it's often applications want to take a totally control on logging, they don't want us use stdout/stderr directly.

Reply:
Makes sense. We can still return `Common::Status` as you suggested below.